AutoRest/Generators/NodeJS/NodeJS/ClientModelExtensions.cs

```diff
@@ -242,6 +242,28 @@ private static string ValidatePrimaryType(this PrimaryType primary, IScopeProvid
             }
         }
 
+        /// <summary>
+        /// Returns the TypeScript type string for the specified primary type
+        /// </summary>
+        /// <param name="primary">primary type to query</param>
+        /// <returns>The TypeScript type correspoinding to this model primary type</returns>
+        private static string PrimaryTSType(this PrimaryType primary) {
+            if (primary == PrimaryType.Boolean)
+                return "boolean";
+            else if (primary == PrimaryType.Double || primary == PrimaryType.Int || primary == PrimaryType.Long)
+                return "number";
+            else if (primary == PrimaryType.String)
+                return "string";
+            else if (primary == PrimaryType.Date || primary == PrimaryType.DateTime)
+                return "Date";
+            else if (primary == PrimaryType.Object)
+                return "Object";   // TODO: test this
```

Comment:
Just for my understanding, is there any difference in the semantics between Object and any as far as the *caller* is concerned?   Object is certainly more restrictive for the callee (which is moot in this case for a d.ts file as the callee isn't type checked), but does it matter to the caller?   I just wanted to make sure I understood the distinction.   Still, any is probably a better choice here for reasons of convention even if the semantics are exactly the same.   So I can change it for that reason.   And thanks for the review.